package.json

```diff
@@ -63,7 +57,6 @@
     "gts": "3.1.0",
     "husky": "5.2.0",
     "jest": "27.5.1",
-    "lerna": "5.6.2",
```

Comment:
Stupid question but did we completely replace lerna by nx ? Was lerna only used for publishing up to this point ?

Reply:
Yeah, Lerna was replaced by NPM Workspaces, Nx and our new process.